commands/azureCommands/delete-image.ts

```diff
@@ -50,28 +73,19 @@ async function removeImage(context: AzureImageTagNode, untag: boolean): Promise<
         image = new AzureImage(repo, wholeName[1]);
     }
 
-    let message: string;
-    let path: string;
-    let digest: string;
-    if (untag) {
-        message = `Are you sure you want to untag: \'${image.toString()}\'? This does not delete the manifest referenced by the tag.`;
-        path = `/v2/_acr/${repo.name}/tags/${image.tag}`;
-    } else {
-        digest = await acrTools.getImageDigest(image);
-        let images = await acrTools.getImagesByDigest(repo, digest);
-        message = `Are you sure you want to delete the manifest: '${digest}' and the associated image(s) ${images.join(', ')}?`;
-        path = `/v2/${repo.name}/manifests/${digest}`;
-    }
+    const digest = await acrTools.getImageDigest(image);
+    const images = await acrTools.getImagesByDigest(repo, digest);
+
+    const shouldDelete = await ext.ui.showWarningMessage(
+        `Are you sure you want to delete the manifest: '${digest}' and the associated image(s) ${images.join(', ')}?`,
+        { modal: true },
+        DialogResponses.deleteResponse,
+        DialogResponses.cancel);
 
-    const shouldDelete = await ext.ui.showWarningMessage(message, { modal: true }, DialogResponses.deleteResponse, DialogResponses.cancel);
     if (shouldDelete === DialogResponses.deleteResponse) {
-        const { acrAccessToken } = await acrTools.acquireACRAccessTokenFromRegistry(registry, `repository:${repo.name}:*`);
-        await acrTools.sendRequestToRegistry('delete', getLoginServer(registry), path, acrAccessToken);
-        if (untag) {
-            vscode.window.showInformationMessage(`Successfully untagged: \'${image.toString()}\'`);
-        } else {
-            vscode.window.showInformationMessage(`Successfully deleted manifest: ${digest}`);
-        }
+        await acrTools.deleteImage(repo, digest);
+        vscode.window.showInformationMessage(`Successfully deleted manifest: ${digest}`);
```

Comment:
Here a colon isn't technically wrong although personally I'd prefer it without.  Up to you.